app/controllers/well_known/oauth_metadata_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module WellKnown
+  class OauthMetadataController < ActionController::Base # rubocop:disable Rails/ApplicationController
+    include CacheConcern
+
+    # Prevent `active_model_serializer`'s `ActionController::Serialization` from calling `current_user`
+    # and thus re-issuing session cookies
```

Comment:
instead of copying and pasting this comment between multiple different files (this is the third file with an identical comment), we should create a self-documenting helper method that calls this function internally. 

![image](https://github.com/mastodon/mastodon/assets/233815/d3931a59-be97-49b5-bed2-aa637969e28d)

Reply:
Any suggestions what to call the helper?

Reply:
Have opened #29249 for this, since it's kinda out-of-scope tech debt that needs further discussion, imo.